scripts/build_synapse_stack.sh

```diff
@@ -26,6 +26,9 @@ PORTAL_BEANSTALK_VERSION=${6}
 # example Blue
 VPC_SUBNET_COLOR=${7}
 
+# example 5.5.0
+BEANSTALK_PLATFORM_VERSION=${8}
+
```

Comment:
I think you need to change the next line to SRC_PATH=${9}

Reply:
How did I miss that?